### PR TITLE
82 improving styling

### LINKED ISSUE #82: Work on improving the styling
## Description
Check font sizing and styles, check colours.

## Plan
- [ ] Check font size
- [ ] Check font styles 
- [ ] Check colours
- [ ] Update code

### PR DESCRIPTION
**Pull request by HedgehogHolly**
Fixes #82 

## Description

- Fixed text size/ratio
- Added Poppins Regular
- Added limegreen colour
- Changed colour of focus
- Added underline hover effect to nav/links
- Changed class name for nav text
- Defined font weights 

🐧

### CODE REVIEW

File: src/styles/app.scss [hunk 1]
```diff
@@ -502,16 +523,15 @@ nav,
...
+  font-weight: 600;
   line-height: 1;
 }
 
 .nav-item--active,
-.nav-item:hover {
-  .nav-item-text {
+.text-selector:hover {
     color: $white;
+    text-decoration: underline;
```
Comment (Nikomus) [inline src/styles/app.scss]: I think in the other places we're using "underlines" they're actually a bottom border.

You could try something like `border-bottom: 2px solid $white;` or similar

File: src/Views/Nav.elm [hunk 2]
```diff
@@ -30,7 +30,7 @@ view _ =
             , span [ class "nav-item" ]
                 [ a [ class "btn ", href "https://google.com", target "_blank", rel "noopener", onClick (Exit True) ]
                     [ getIcon "exit-door" (Just "nav-item-text nav-icon")
-                    , span [ class " nav-item-text" ] [ text "Exit Site" ]
+                    , span [ class " nav-item-text text-selector" ] [ text "Exit Site" ]
```
Comment (Nikomus) [inline src/Views/Nav.elm]: Can we give this class a name that will help identify what kind of text it's selecting?

File: src/styles/app.scss [hunk 3]
```diff
@@ -52,7 +70,7 @@ $max-width: 1044px;
 }
 
 *:focus {
-  outline-color: $secondary-colour;
+  outline-color: $lime-green;
```
Comment (Nikomus) [inline src/styles/app.scss]: Do you want this to be the `$focus-colour` you defined?

Comment (HedgehogHolly) [reply src/styles/app.scss]: yeah, I couldn't get it to work for some reason

File: src/Views/Nav.elm (hunk 2 above)
Comment (HedgehogHolly) [reply src/Views/Nav.elm]: sure, nav-text-selector maybe?

File: src/styles/app.scss (hunk 1 above)
Comment (HedgehogHolly) [reply src/styles/app.scss]: Ah okay, that makes more sense. I can change it to that 👍

**Comment by HedgehogHolly** [comment]
I think I've made all the fixes @Nikomus 🐧

**Review by Nikomus — APPROVED** [review]
Awesome! Give it a merge 🌯

**Pull request merged**